iaso/tests/api/test_profile_logs.py

```diff
@@ -0,0 +1,542 @@
+import datetime
+import jsonschema
+import pytz
+from hat.audit.models import Modification
+from iaso.test import APITestCase
+from hat.menupermissions.constants import MODULES
+from iaso import models as m
+from django.contrib.contenttypes.models import ContentType
+from hat.menupermissions import models as permission
+from unittest.mock import patch
+
+user_schema = {
+    "type": "object",
+    "properties": {
+        "user_id": {"type": "number"},
+        "username": {"type": "string"},
+        "first_name": {"type": "string"},
+        "last_name": {"type": "string"},
+    },
+    "required": ["user_id", "username", "first_name", "last_name"],
+}
+
+location = {
+    "type": "object",
+    "properties": {"id": {"type": "number"}, "name": {"type": "string"}},
+    "required": ["id", "name"],
+}
+
+PROFILE_LOG_LIST_SCHEMA = {
```

Comment:
Since you did some checks with a json schema, I guess there is no need, but otherwise you could have tested each serializer to make sure they are properly sending data, like in `iaso.tests.api.org_unit_change_requests.test_serializers.py`

Reply:
I thought about it, but then, there's the jsoncshema validation + I test all fields in other tests so I thought it should be covererd